src/layout/Auth/AuthContainer.jsx

```diff
@@ -8,28 +8,22 @@ import {
   AuthStates,
 } from 'redux/actions.js';
 import logger from 'logger.js';
-import Auth from './Auth.jsx';
+import { Auth } from './components';
 
 class AuthContainer extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      logErr: '',
-      regErr: '',
-      waiting: false,
-    };
-
-    /*
-      Only functions that are being passed out of this scope, as in the case
-      of passing props to children, need to be bound. This ensures that they
-      can still access the scope of this component even though they are
-      being called from another component.
-    */
-    this.handleRegister = this.handleRegister.bind(this);
-    this.handleLogin = this.handleLogin.bind(this);
-  }
-
-  handleRegister(data) {
+  state = {
```

Comment:
Is this the preferred way of doing things these days? Moving state and binding out of the constructor, that is

Reply:
Yes, it is